src/Element.js

```diff
@@ -11,6 +11,12 @@ var querySelectorAll = require('query-selector')
 function styleCamelCase (name) {
   var camel = camelCase(name)
 
+  // Detect if the style property is already camelCased
+  // To not convert Webkit*, Moz* and O* to lowercase
+  if (camel.slice(1) === name.slice(1)) {
+    return name
+  }
+
   if (name[0] === '-') {
     return camel[0].toUpperCase() + camel.slice(1)
   } else {
```

Comment:
I thought this issue is what I attempted to solve on the line above, I guess it doesn't work because it doesn't start with a hyphen in this case but an upper-case character? I do wonder if we can use this instead.

Why not check if the first character is a hyphen or an upper-case character in this if statement? It would give you the same result, you could use a nice little regular expression too :)